README.md

```diff
@@ -14,18 +14,48 @@ files for assignments that require GatorGradle. ConfiGator uses
 
 ## Usage
 
-### Installing Python dependencies
+### Installing ConfiGator
 
-In order to install the dependencies of this project, run this command in your
-command line:
+Command to install ConfiGator
 
 ```bash
-poetry install
+pipx install configator
 ```
 
-After installing the dependencies, navigate to the repository directory, and run
-the following in your command line:
+### Commands to Change Specific Information Within a File
+
+Command to change name
 
 ```bash
-poetry run python main.py
+configator --help
 ```
+
+This command will show a list of every available command to change specific information
+within a file of your choice. The list will include commands to change the
+name of the assignment, change if gradle run breaks if there is a check failure,
+change if gradle run should immediately break upon a check failure through fastfail,
+change the indentation the body of the file should use, and the GatorGrader version
+of the assignment.
```

Comment:
Although additional detail is normally good, it may not be a good idea to list every possible configuration change that can be made in english here; instead, simply say something like "The list will include commands to change various configuration that is written to `config/gatorgrader.yml`, or `build.gradle`".